web-common/src/features/entity-management/file-content-utils.spec.ts

```diff
@@ -27,25 +27,13 @@ describe("parseKindAndNameFromFile", () => {
     [
       "explicit kind and name for yaml",
       "sources/AdBids_dashboard.yaml",
-      `\n\nkind : metrics_view\nname:AdBids_dashboard_name\nmodel: AdBids_model\nmeasures: []\ndimensions: []`,
+      `\n\ntype : metrics_view\nname:AdBids_dashboard_name\nmodel: AdBids_model\nmeasures: []\ndimensions: []`,
       { kind: ResourceKind.MetricsView, name: "AdBids_dashboard_name" },
     ],
-    [
-      "explicit kind and name for sql",
```

Comment:
Why are we removing tests for .sql?

Reply:
Because we're now assuming all SQL files are models and we aren't documenting `--@kind: model` anywhere, I figured there's no need to support it in these tests.

Reply:
Shouldnt we remove the code as well then? Also there is a test for `@name` as well in there. IMO we should keep this since backend didnt remove the support for it.

Reply:
Makes sense, just restored.